### DISCUSSION TITLE
RxJS for Typescript

### DISCUSSION THREAD

**Discussion by dshyryayev** [ReactiveX/rxjs#7153] [Q&A] [answered]
I was trying to find it, may it does exists?
Any plans for Typescript?

**Comment by kwonoj** [comment, marked as answer]
RxJS is written in typescript and support typescript types with built-in types. What does `plans for typescript` exactly means?

**Reply by dshyryayev** [reply]
Yes, you are right @kwonoj nothing else needs to be done. Thank you!
